pkg/querier/blocks_store_queryable.go

```diff
@@ -944,12 +949,11 @@ func convertBlockHintsToULIDs(hints []hintspb.Block) ([]ulid.ULID, error) {
 	return res, nil
 }
 
-func countSeriesBytes(series []*storepb.Series) (count uint64) {
+// countChunkBytes returns the size of the chunks making up the provided series in bytes
+func countChunkBytes(series ...*storepb.Series) (count int) {
 	for _, s := range series {
 		for _, c := range s.Chunks {
-			if c.Raw != nil {
-				count += uint64(len(c.Raw.Data))
-			}
+			count += c.Size()
```

Comment:
It's not the exact same as `len(c.Raw.Data)`. I would get stick to `len(c.Raw.Data)` which is the actual chunk size and not the whole struct size. `len(c.Raw.Data)` is also faster to run.

Reply:
Right, I was a little confused about which I should use. The limit is applied using the whole struct size (`chunk.Size()`). My thinking was that the metric should reflect the value that the limit is using. Do you think I should change the way the limit is applied above to only consider `c.Raw.Data` and change everything to use that value?

Reply:
Good point! I checked the code again. Use `.Size()` and ignore my previous message. Why? Right now there's no real difference but in the future we'll add Thanos downsampling support than `.Size()` is more correct because that's size of the chunk fetched (including all downsampled versions).